### ISSUE TITLE
threshold value in config.ts seems to be ignored

### ISSUE THREAD

**Issue by schildbach** [hsjoberg/fork-explorer#52]
I'm passing in this config.ts snippet. These are the values for testnet.

```
  fork: {
    name: "Taproot",
    info: [],
    versionBit: 2,
    threshold: 1512,
  },
```

But on the rendered overview page, it shows 90% at several places. I'd expect something like 75%.

**Comment by hsjoberg** [comment]
Hmm, yeah it's hardcoded in some places.

**Comment by hsjoberg** [comment]
@schildbach addressed in 97a0bab.
